### ISSUE TITLE
InfluxDB stops every couple of hours, with no visible reason 2

### ISSUE THREAD

**Issue by MindFreeze** [hassio-addons/addon-influxdb#97]
### TLDR 
Storage was too slow to handle the data

# Problem/Motivation

InfluxDB stopping and therefore I lose my data, as I'm not always aware of the stop. See logs from #72 

## Expected behavior

I want my InfluxDB to run without issues.

## Actual behavior

InfluxDB is running for some time, and then randomly stops without notice.

## Steps to reproduce

Run InfluxDB, act normal & "forget about HomeAssistant" as the automations do stuff at home. then, after few hours check the supervisor and see that InfluxDB stopped working some time ago.

## Proposed changes

Maybe mention this in docs somewhere

## Cause and solution

I was using an SSD through a USB2 to SATA adapter for all my HA data. HA running on rPi4 in docker over raspbian. 
Influxdb addon kept stopping without notice. RAM was fine, CPU was fine. 
Totally unrelated, I upgraded my storage to a faster SSD and a USB3 to M.2 NVME adapter. Influxdb has been running for a week now without fail. CPU usage has doubled, but that is normal for the amount of data, I guess.

I think installing influxdb on the bare metal worked for the OP in #72 because it just skipped a lot of file system overhead. Docker has some IO overhead, so he basically optimized it by skipping docker.

Feel free to close this issue. I just thought it should be documented for posterity.

**Comment by frenck** [comment]
Well, to begin with, you are running an unsupported system.
Nothing stops without trace, there is always something.

This story does not add anything. Please provide proper debugging and tracing information, without it, I cannot do anything.
